src/memgraph.cpp

```diff
@@ -435,11 +435,6 @@ int main(int argc, char **argv) {
     db_config.durability.snapshot_wal_mode = DISABLED;
   }
 
-  if (memgraph::flags::AreExperimentsEnabled(memgraph::flags::Experiments::VECTOR_SEARCH) &&
-      db_config.salient.storage_mode == memgraph::storage::StorageMode::ON_DISK_TRANSACTIONAL) {
```

Comment:
Under `ON_DISK_TRANSACTIONAL` the vector search is still not working -> revert back that part of the if